public/app/plugins/datasource/elasticsearch/configuration/utils.ts

```diff
@@ -1,24 +1,18 @@
-import { valid } from 'semver';
-
 import { DataSourceSettings } from '@grafana/data';
 
 import { ElasticsearchOptions } from '../types';
-import { coerceESVersion } from '../utils';
 
-import { defaultMaxConcurrentShardRequests } from './ElasticDetails';
```

Comment:
`defaultMaxConcurrentShardRequests` is still available in `ElasticDetails`, but it is not used anywhere else.

Reply:
thanks, will keep it in mind to clean it up ,when i progress with this draft pr 👍